doc/src/man/litani-set-jobs.scdoc

```diff
@@ -0,0 +1,77 @@
+litani-set-jobs(1) "" "Litani Build System"
+
+; Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+; Licensed under the Apache License, Version 2.0 (the "License").
+; You may not use this file except in compliance with the License.
+; A copy of the License is located at
+
+;     http://www.apache.org/licenses/LICENSE-2.0
+
+; or in the "license" file accompanying this file. This file is distributed
+; on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+; express or implied. See the License for the specific language governing
+; permissions and limitations under the License.
+
+
+# NAME
+
+litani set-jobs - Set the list of jobs to be run
+
+# SYNOPSIS
+
+*litani set-jobs*
+	\[*-f*/*--from-file* _F_]
+	\[*-s*/*--from-string* _S_]
+
+
+# DESCRIPTION
+
+This program reads a list of jobs as a JSON list. Each job in the list will be
+a dict with keys being the flags to *litani add-job(1)* with dashes replaced by
+underscores. This list of jobs will replace those in the current build graph.
```

Comment:
I think with my new implementation of set-jobs it shouldn't really matter if users pass in dashes or underscores

Reply:
Then in the documentation let's suggest the easy thing. I think dashes are easier than underscore, first of all because of the shift key, and also because it's closer to the actual command-line name

Reply:
If the user is getting the list of jobs from get-jobs, then they will be underscores already